politeiawww/cmd/politeiawwwcli/commands/editproposal.go

```diff
@@ -13,6 +13,58 @@ import (
 	"github.com/decred/politeia/util"
 )
 
+// Help message displayed for the command 'politeiawwwcli help editproposal'
+var EditProposalCmdHelpMsg = `editproposal "token" "markdownFile" "attachmentFiles" 
+
+Edit a proposal.
+
+Arguments:
+1. token             (string, required)   Proposal censorship token
+2. markdownFile      (string, required)   Edited proposal 
+3. attachmentFiles   (string, optional)   Attachments 
+
+Result:
+{
```

Comment:
Same comment as above about differentiating between the request body and the response.  Apply this to all the commands that have a request body.